client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -51,12 +57,146 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> Analysis,
         isComment -> Comment,
         tagExistsWithId("tone/features") -> Feature,
-        liveBlog -> Live,
-        deadBlog -> Article
+        isLiveBlog -> Live,
+        isDeadBlog -> Article
       )
 
-      val result = predicates.collectFirst { case (predicate, design) if predicate(content) => design }
+      val result = getFromPredicate(content, predicates)
       result.getOrElse(defaultDesignType)
     }
   }
+
+  implicit class RenderingFormat(val content: Content) extends AnyVal {
+
+
+    def design: Design = {
+
+      val defaultDesign: Design = ArticleDesign
+
+      val isPhotoEssay: ContentFilter = content => content.fields.flatMap(_.displayHint).contains("photoessay")
+
+      val isInteractive: ContentFilter = content => content.`type` == ContentType.Interactive
+
+      val predicates: List[(ContentFilter, Design)] = List(
+        tagExistsWithId("artanddesign/series/guardian-print-shop") -> PrintShopDesign,
+        isMedia -> MediaDesign,
+        isReview -> ReviewDesign,
+        tagExistsWithId("tone/analysis") -> AnalysisDesign,
+        tagExistsWithId("tone/comment") -> CommentDesign,
+        tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("tone/features") -> FeatureDesign,
+        tagExistsWithId("tone/recipes") -> RecipeDesign,
+        tagExistsWithId("tone/matchreports") -> MatchReportDesign,
+        tagExistsWithId("tone/interview") -> InterviewDesign,
+        tagExistsWithId("tone/editorials") -> EditorialDesign,
+        tagExistsWithId("tone/quizzes") -> QuizDesign,
+        isInteractive -> InteractiveDesign,
+        isPhotoEssay -> PhotoEssayDesign,
+        isLiveBlog -> LiveBlogDesign,
+        isDeadBlog -> DeadBlogDesign
+      )
+
+      val result = getFromPredicate(content, predicates)
+      result.getOrElse(defaultDesign)
+    }
+
+    def theme: Theme = {
+      val defaultTheme: Theme = NewsPillar
+
+      val specialReportTags: Set[String] = Set(
+        "business/series/undercover-in-the-chicken-industry",
+        "business/series/britains-debt-timebomb",
+        "world/series/this-is-europe",
+        "environment/series/the-polluters",
+        "news/series/hsbc-files",
+        "news/series/panama-papers",
+        "us-news/homan-square",
+        "uk-news/series/the-new-world-of-work",
+        "world/series/the-new-arrivals",
+        "news/series/nauru-files",
+        "us-news/series/counted-us-police-killings",
+        "australia-news/series/healthcare-in-detention",
+        "society/series/this-is-the-nhs"
+      )
+
+      def isPillar(pillar: String): ContentFilter = content => content.pillarName.contains(pillar)
+
+      val isSpecialReport: ContentFilter = content => content.tags.exists(t => specialReportTags(t.id))
```

Comment:
What happens for special reports assigned in the targeting tool?

Reply:
Hmm, I did this off the back of different decision functions I found across the different codebases. If there's one missing, then we'll need to implement it in a future version. I imagine the process of integrating this all into the platforms will highlight missing areas and then we can update this to reflect that .